packages/vhd-lib/Vhd/VhdDirectory.js

```diff
@@ -174,6 +174,10 @@ exports.VhdDirectory = class VhdDirectory extends VhdAbstract {
     return `blocks/${blockPrefix}/${blockSuffix}`
   }
 
+  getFullBlockPath(blockId) {
+    return this._getChunkPath(this._getBlockPath(blockId))
+  }
```

Comment:
I think we should keep this private and implement a `moveBlock` instead which will handle the action depending of the VHD implementation.